Library/Formula/libtbox.rb

```diff
@@ -0,0 +1,46 @@
+class Libtbox < Formula
+  desc "The Cross-platform C development library"
+  homepage "https://github.com/waruqi/tbox"
+  url "https://github.com/waruqi/tbox/archive/v1.5.1.tar.gz"
+  mirror "http://tboox.net/release/tbox/tbox-v1.5.1.tar.gz"
+  sha256 "0bd5f88482cb96de5e1a488f1dc20fd759b5a00e992762915f4c8432276462ef"
+  head "https://github.com/waruqi/tbox.git"
+
+  option "with-debug", "Build with debug support"
+  option "without-zlib", "Disable Zlib compression library"
+  option "without-sqlite", "Disable Sqlite database"
```

Comment:
Can this not use the system `sqlite` at all? It has to use ours, even on newer platforms?